CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 BUG FIXES:
 
 * `Operator`: fix an issue when the operator couldn't be run on the `amd64` platform.
+* `AgentPool`: add `autoscaling` field to allow configuration of a basic autoscaler for agent deployments based on pending runs. 
```

Comment:
Maybe `ENHANCEMENT`? :-)
And please, add the `[[GH-198](https://github.com/hashicorp/terraform-cloud-operator/pull/198)]` reference. The one above is already fixed and in the main branch.

All in all looks good, approved! Thank you!